generators/generator-base.js

```diff
@@ -46,8 +46,8 @@ Generator.prototype.addElementToMenu = function (routerName, glyphiconName, enab
             file: fullPath,
             needle: 'jhipster-needle-add-element-to-menu',
             splicable: [
-                '<li ui-sref-active="active" >\n' +
-                '                    <a ui-sref="' + routerName + '" ng-click="vm.collapseNavbar()">\n' +
+                '<li uiSrefActive="active" >\n' +
```

Comment:
These 3 methods are used by ng1 client generator and hence would break those. We need to add new methods or if else blocks for ng2 client. But its required only for entity creation which we havent migrated yet, hence we can change these methods later